### PR TITLE
index: fix 4 links

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

Fix 4 broken links.

### Which version does your change affect? 

Just _index.md

@dcalvin @lilin90 PTAL

### CODE REVIEW

File: _index.md [hunk 1]
```diff
@@ -47,10 +47,10 @@ TiDB 对业务没有任何侵入性，能优雅的替换传统的数据库中间
 
 TiDB 可以部署在本地和云平台上，支持公有云、私有云和混合云。你可以根据实际场景或需求，选择相应的方式来部署 TiDB 集群：
 
-- [使用 Ansible 部署](/how-to/deploy/orchestrated/ansible.md)：如果用于生产环境，须使用 Ansible 部署 TiDB 集群。
-- [使用 Ansible 离线部署](/how-to/deploy/orchestrated/offline-ansible.md)：如果部署环境无法访问网络，可使用 Ansible 进行离线部署。
-- [使用 Docker Compose 部署](/how-to/get-started/local-cluster/install-from-docker-compose.md)：如果你只是想测试 TiDB、体验 TiDB 的特性，或者用于开发环境，可以使用 Docker Compose 在本地快速部署 TiDB 集群。该部署方式不适用于生产环境。
-- [使用 Docker 部署](/how-to/deploy/orchestrated/docker.md)：你可以使用 Docker 部署 TiDB 集群，但该部署方式不适用于生产环境。
+- [使用 Ansible 部署](/dev/how-to/deploy/orchestrated/ansible.md)：如果用于生产环境，须使用 Ansible 部署 TiDB 集群。
```
Comment (lilin90) [inline _index.md]:
```suggestion
- [使用 Ansible 部署](/v3.0/how-to/deploy/orchestrated/ansible.md)：如果用于生产环境，须使用 Ansible 部署 TiDB 集群。
```

File: _index.md [hunk 2]
```diff
@@ -47,10 +47,10 @@ TiDB 对业务没有任何侵入性，能优雅的替换传统的数据库中间
...
+- [使用 Ansible 离线部署](/dev/how-to/deploy/orchestrated/offline-ansible.md)：如果部署环境无法访问网络，可使用 Ansible 进行离线部署。
```
Comment (lilin90) [inline _index.md]:
```suggestion
- [使用 Ansible 离线部署](/v3.0/how-to/deploy/orchestrated/offline-ansible.md)：如果部署环境无法访问网络，可使用 Ansible 进行离线部署。
```

File: _index.md [hunk 3]
```diff
@@ -47,10 +47,10 @@ TiDB 对业务没有任何侵入性，能优雅的替换传统的数据库中间
...
+- [使用 Docker Compose 部署](/dev/how-to/get-started/local-cluster/install-from-docker-compose.md)：如果你只是想测试 TiDB、体验 TiDB 的特性，或者用于开发环境，可以使用 Docker Compose 在本地快速部署 TiDB 集群。该部署方式不适用于生产环境。
```
Comment (lilin90) [inline _index.md]:
```suggestion
- [使用 Docker Compose 部署](/v3.0/how-to/get-started/local-cluster/install-from-docker-compose.md)：如果你只是想测试 TiDB、体验 TiDB 的特性，或者用于开发环境，可以使用 Docker Compose 在本地快速部署 TiDB 集群。该部署方式不适用于生产环境。
```

File: _index.md [hunk 4]
```diff
@@ -47,10 +47,10 @@ TiDB 对业务没有任何侵入性，能优雅的替换传统的数据库中间
...
+- [使用 Docker 部署](/dev/how-to/deploy/orchestrated/docker.md)：你可以使用 Docker 部署 TiDB 集群，但该部署方式不适用于生产环境。
```
Comment (lilin90) [inline _index.md]:
```suggestion
- [使用 Docker 部署](/v3.0/how-to/deploy/orchestrated/docker.md)：你可以使用 Docker 部署 TiDB 集群，但该部署方式不适用于生产环境。
```

**Comment by lilin90** [comment]
@YiniXu9506 Four deployment links on this page (https://pingcap.com/docs-cn/) are broken. We tried to fix them by adding `/v3.0`, but it seems it does not work. Is there something wrong with this?

**Comment by YiniXu9506** [comment]
@lilin90 @CaitinChen The links are woking online. PTAL

**Comment by lilin90** [comment]
@YiniXu9506 Yes. It works well now. Thanks!

**Pull request merged**